features/step_definitions/kata.rb

```diff
@@ -41,7 +41,7 @@
 end
 
 Given /^I wait until number of completed kata runtime nodes match#{OPT_QUOTED} for #{QUOTED}$/ do |number, kc_name|
-  ready_timeout = 900
+  ready_timeout = 1000
```

Comment:
this is ~16 minutes, which should be enough for the typical cluster sizes we get in our public cloud clusters. Not sure if this will ever run on bare-metal clusters. If yes, this could be not enough depending on how many nodes the machine pool has.

Reply:
@jensfr yeah for our acceptance test, I saw a couple of times 900 seconds was not enough, that's why I extended it to 1000 seconds.  I believe that should be sufficient.